### DISCUSSION TITLE
Dịch baseline (model)

### DISCUSSION THREAD

**Discussion by cuongvng** [mlbvn/handson-ml2-vn#72] [Terminology] [answered]
Từ này mình chưa thấy có trong glossary, đề xuất là `mô hình cơ sở`.
Mọi người nghĩ sao? @mlbvn/handson-ml2

**Comment by minhduc0711** [comment]
Mình thấy ổn, vừa đào lại [thread này](https://github.com/mlbvn/d2l-vn/pull/1156#discussion_r407895453) thì có thêm từ `nền` nữa.

**Reply by cuongvng** [reply]
có vẻ là đang bị nhập nhằng việc baseline có "cố định" hay không. Theo [ML Blog @CMU](https://blog.ml.cmu.edu/2020/08/31/3-baselines/) thì có 2 loại:

> We will see that there are two different types of baselines, one which refers to a simple model, and another which refers to the best model from previous works

Có lẽ nên giữ nguyên?

**Reply by quangnhat185** [reply]
Kể cả có cố định hay không thì mình vẫn dịch là `cơ sở` được mà. Cũng mang ý nghĩa chung là một nền để so sánh, còn việc lựa chọn nền thì có thể linh hoạt

**Reply by tiepvupsu** [reply]
`base` = `nền`?

**Reply by minhduc0711** [reply]
Em vote `cơ sở` vì nghe hay hơn 1 chút.

**Comment by thanhcsf** [comment, marked as answer]
Điểm này tạm chốt là **`Cơ sở`** nha anh em.
